### PR TITLE
Created release notes for June 2015 release

### PR DESCRIPTION
**Pull request by csharpfritz**
The release is referred to as 3.0RC2 in this documentation

### CODE REVIEW

File: NuGet.Docs/Release-Notes/NuGet-3.0-RC2.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+# NuGet 3.0 RC2 Release Notes
+
+[NuGet 3.0 RC Release Notes](nuget-3.0-RC)
+
+NuGet 3.0 RC2 was released on June 3, 2015 as an interim release available from the Visual Studio 2015 Extension Gallery and Codeplex. This release has a number of important bug fixes and performance improvements that we felt were important to release before the completed Visual Studio 2015 release. This NuGet extension version is only available for Visual Studio 2015.
+
+In total, we closed 158 issues in this release, and you can review the [complete list of issues on GitHub](https://github.com/NuGet/Home/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A3.0.0-RTM+sort%3Aupdated-asc+updated%3A%3C%3D2015-06-01).
+
+## Summary of issues resolved
```
Comment (yishaigalatzer) [inline NuGet.Docs/Release-Notes/NuGet-3.0-RC2.md]: Summary top issues resolved?

**Comment by yishaigalatzer** [comment]
Add a download link to codeplex.

fix, squash and commit as soon as @feiling puts the extension on codeplex.

**Comment by csharpfritz** [comment]
Updated as requested and squashed

**Comment by yishaigalatzer** [comment]
:ship it:

**Pull request merged**